src/main/scala/edu/ie3/simona/model/participant2/PvModel.scala

```diff
@@ -74,25 +74,30 @@ class PvModel private (
   private val activationThreshold =
     sRated.toActivePower(cosPhiRated) * 0.001 * -1
 
-  override val initialState: ModelInput => PvState = { input =>
-    val weatherData = getWeatherData(input.receivedData)
-
-    PvState(
-      input.currentTick,
-      input.currentSimulationTime,
-      weatherData.diffIrr,
-      weatherData.dirIrr,
-    )
+  override val initialState: (Long, ZonedDateTime) => PvState = {
```

Comment:
The brace is unnecessary.

Reply:
`val` functions always look a bit more complicated to me without braces, but okay, let's try without